org.mixedrealitytoolkit.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -28,12 +28,43 @@ public class GazePinchInteractor :
         [Tooltip("The hand controller used to get the selection progress values")]
         private ArticulatedHandController handController;
 
+        #region Properties and methods from deprecated XRBaseController
+
+        XRControllerState m_ControllerState;
+        /// <summary>
+        /// The current state of the controller.
+        /// </summary>
+        public XRControllerState currentControllerState
+        {
+            get
+            {
+                SetupControllerState();
+                return m_ControllerState;
+            }
+
+            set
+            {
+                m_ControllerState = value;
+                m_CreateControllerState = false;
+            }
+        }
+
+        bool m_CreateControllerState = true;
+
+        void SetupControllerState()
```

Comment:
style....put functions down below, near other private functions.   And explicitly mark this as "private"

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/182e3c8900231046fe4cedb5ecdcb21797f6f946 , resolving.